terraform/main.tf

```diff
@@ -165,7 +165,7 @@ module "aws_deploy-dev2-eu-west-2" {
 module "aws_deploy-integration-eu-west-2" {
   source            = "modules/cloud/aws/deploy"
   env               = "integration"
-  bootstrap_version = "master"
```

Comment:
I see dev1 and dev2 still master. Intended?

Reply:
Yes, I'd like to have an option to test the master via dev environments, while I usually spin up new environment, I'd like to keep this option.